lib/styles.js

```diff
@@ -96,7 +96,11 @@ module.exports = function (expect) {
         var isSymbol;
         isSymbol = typeof key === 'symbol';
         if (isSymbol) {
-            this.text('[').sp().appendInspected(key).sp().text(']').text(':');
+            if (isArrayLike) {
+                this.appendInspected(key).text(':');
```

Comment:
Hmm, why render it without the `[`...`]` in an array? Both `[ Symbol('foo'): 123 ]` and `[ [ Symbol('foo') ]: 123 ]` are invalid JavaScript syntax anyway, so we might as well always use the latter?

Reply:
Hmm fair point - I think my only reasoning was that it looked a little weird, particularly in the inspection line, to have extra square brackets and I wasn't sure about the possiblity for some confusion as compared to nested arrays. But I'll totally follow your lead on this :)

Reply:
Mhh aren't we talking about objects here: `x = { [ Symbol('foo') ]: 123 }`?

Reply:
I think we should go with `[ [Symbol('foo')]: 123 ]` as it is closer to the valid object syntax.